MapboxAndroidDemo/proguard-rules.pro

```diff
@@ -121,13 +121,17 @@
 -keep class android.arch.lifecycle.** { *; }
 -keep class com.mapbox.android.core.location.** { *; }
 -keep class com.mapbox.mapboxsdk.** { *; }
```

Comment:
This shouldn't be necessary because we're using the `@Keep` annotation in the Maps SDK.